cmd/ui/src/views/Explore/utils.ts

```diff
@@ -38,26 +40,30 @@ export const initGraphNodes = (graph: MultiDirectedGraph, nodes: GraphNodes, nod
 
         // Tier zero nodes should be marked with a gem glyph
         if (node.isTierZero) {
-            const glyph = GLYPHS[GlyphKind.TIER_ZERO];
+            const glyph =
+                theme.palette.color.primary === '#1D1B20'
```

Comment:
Can't help but feel like this logic will break and go undetected for a while.

Maybe there is a more direct way to determine if the app is in dark mode or not (perhaps via another argument to this function?)

Reply:
Yea I agree and had forgotten to clean this up so thank you for pointing it out. I am going to pass in the dark mode state directly instead.